linux/system.go

```diff
@@ -83,6 +84,25 @@ func (ls *linuxSystem) ScanDisks(filter disko.DiskFilter,
 	return disks, nil
 }
 
+func getDiskReadOnlyProperty(d disko.UdevInfo) (bool, error) {
+	syspath, err := getSysPathForBlockDevicePath(d.Name)
+	if err != nil {
+		return false, err
+	}
+
+	syspathReadOnly := syspath + "/ro"
+	if disko.PathExists(syspathReadOnly) {
+		content, err := ioutil.ReadFile(syspathReadOnly)
```

Comment:
ioutil.ReadFile will probably return a NotExist error, so you don't really have to check first.

Reply:
You're right, I'll fix.